lib/block_view_dao_coin_limit_order.go

```diff
@@ -236,6 +246,9 @@ func (bav *UtxoView) _getNextLimitOrdersToFill(
 			if err != nil {
 				return nil, err
 			}
+		} else {
+			// TODO: switch to RuleErrorDAOCoinLimitOrderUnsupportedOperationType
+			return nil, fmt.Errorf("Invalid operation type")
```

Comment:
```suggestion
			return nil, RuleErrorDAOCoinLimitOrderUnsupportedOperationType
```

Reply:
This isn't defined yet. It's defined in https://github.com/deso-protocol/core/pull/253.